scripts/compile-runtime-agents.sh

```
@@ -789,6 +831,57 @@ write_gemini_adapter() {
+    printf 'name: %s\n' "${role}"
+    printf 'description: %s\n' "${canonical_desc}"
+    printf 'canonical_source: %s\n' "${canonical_src}"
+    printf 'canonical_sha: %s\n' "${canonical_sha}"
+    printf 'generator: %s\n' "${GENERATOR_PATH}"
+    printf 'generator_version: %s\n' "${GENERATOR_VERSION}"
+    printf 'classification: generated\n'
+    printf -- '---\n'
+    printf '\n'
+    # shellcheck disable=SC2016  # literal backticks for Markdown output
+    printf '## Goal\n'
+    printf 'Act as the `%s` specialist on the sw-dev-team.\n' "${role}"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)